dashboard/app/views/plc/enrollment_evaluations/preview_assignments.html.haml

```diff
@@ -7,29 +7,34 @@
 = form_tag(action: 'confirm_assignments') do
   .preview_evaluation
     .module_type_section
-      %h3 Required Content Module
+      %h3
```

Comment:
If there aren't multiple things nested, I prefer that they appear on the same line

`%h3= t('plc.preview_assignments.content_focus_header')`